README.md

```diff
@@ -37,3 +37,4 @@ See [`CONTRIBUTING.md`](CONTRIBUTING.md).
 * How do I handle `multipart/form-data` file uploads? ([Scala example](./file-upload/file-upload-scala-sbt/README.md))
 * How do I use a custom message serializer and response header to implement file downloads? ([Scala example](./file-download/file-download-scala-sbt/README.md))
 * How do I integrate Lightbend Telemetry (Cinnamon)? ([Java/Maven example](./lightbend-telemetry/lightbend-telemetry-java-mvn/README.md))
+* How do I configure the Circuit Breaker Panel for a Lagom Java application? ([Java/Maven example](./circuitbreakerpanel/circuitbreakerpanel-java-mvn/README.md))
```

Comment:
Ah! Good addition! 

(I always forget about the main index)

Reply:
About that, last week I merged https://github.com/lagom/lagom-recipes/tree/master/lightbend-telemetry/log-correlation-java-mvn and forgot to add it to the main `README` index too. If you could improve this PR with that info it'd be ❤️ .

Reply:
```markdown
How do I use MDC-Log correlation IDs with Lightbend Telemetry (Cinnamon) in Lagom? ([Java/Maven example](./lightbend-telemetry/log-correlation-java-mvn/README.md))

Reply:
created https://github.com/lagom/lagom-recipes/issues/46